libbeat/docs/upgrading.asciidoc

```diff
@@ -348,3 +367,44 @@ You might be writing to an index that doesn't match the index pattern used
 by {beats}. See <<non-es-outputs>>.
 
 
+[[dashboard-shard-failed]]
+==== "shard failed" error when viewing {beats} dashboards in {kib}
+
+After upgrading to {es} 7.0, any indices created by Beats 6.6 or older will not
+work in {kib} dashboards until the `index.query.default_field` setting is added
+to each index. Indices created in Beats 6.7 or later are unaffected.
+
+*Background:* Starting in {es} 7.0, some query types, such as Simple Query
```

Comment:
@dover As someone new to this problem, I couldn't understand it very well from your description, so I added a bit more background/context based on other docs where we discuss this setting. I hope it works OK because this is not a great end user experience, especially for the subset of Beats users who just want to get things running as fast as possible so they can view the same log data they could see before upgrading. :-(